src/main/groovy/com/github/spotbugs/snom/SpotBugsBasePlugin.java

```diff
@@ -115,20 +115,12 @@ Properties loadProperties() {
   }
 
   private Configuration createPluginConfiguration(Project project) {
-    Configuration configuration =
-        project
-            .getConfigurations()
-            .create(SpotBugsPlugin.PLUGINS_CONFIG_NAME)
-            .setDescription("configuration for the external SpotBugs plugins")
-            .setVisible(false)
-            .setTransitive(true);
-    project
+    return project
         .getConfigurations()
-        .create(SpotBugsPlugin.INTERNAL_CONFIG_NAME)
-        .setDescription(
-            "configuration for the external SpotBugs plugins excluding transitive dependencies")
+        .create(SpotBugsPlugin.PLUGINS_CONFIG_NAME)
+        .setDescription("configuration for the external SpotBugs plugins")
+        .setVisible(false)
```

Comment:
The internal configuration (`actualSpotbugsPlugins`) is needless because the FindBugs plugin is always distributed with its dependencies so no need to add transient dependencies to `spotbugsClasspath`. See [`PluginLoader`](https://github.com/spotbugs/spotbugs/blob/3cce1f0c8e9599e60c1b35030a084435b3c17a0a/spotbugs/src/main/java/edu/umd/cs/findbugs/PluginLoader.java) for detail, it uses `Class-Path` metadata of jar file to find dependencies.